config/routes.rb

```diff
@@ -6,9 +6,7 @@
   resources :activities
   resources :newsletters
 
-  match '/cards/update' => 'cards#update', :via => [:put]
-  match '/cards/:user_slug/edit' => 'cards#edit', :via => [:get]
-  match '/subscriptions/upgrade' => 'subscriptions#upgrade', :via => [:put]
```

Comment:
So. What is the `subscriptions` resource now? Does it have a more appropriate name after the refactor?